### PR TITLE
new commit for Hacktoberfest

### PR DESCRIPTION
**Pull request by code4oryou**
## What does this PR do?
Added a new node.js book for professional with clickable link.

### Why is this valuable (or not)?
The book is for node.js professionals thus will really be helpful for the experienced individuals
### How do we know it's really free?
All we have to do is just visit the website and download its free PDF.

### For book lists, is it a book? For course lists, is it a course? etc.
Yes, it is a book.
## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [x] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [x] Include author(s) and platform where appropriate.
- [x] Put lists in alphabetical order, correct spacing.
- [x] Add needed indications (PDF, access notes, under construction).
- [x] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

**Review by Navninja7 — COMMENTED** [review]
Hello :) Please make sure the links are in alphabetical order, then your checks will pass.

File: books/free-programming-books-subjects.md [hunk 1]
```diff
@@ -732,6 +732,7 @@ Kerridge (PDF) (email address *requested*, not required)
 * [Elasticsearch: The Definitive Guide](https://www.elastic.co/guide/en/elasticsearch/guide/current/index.html) ([fork it on GH](https://github.com/elastic/elasticsearch-definitive-guide))
 * [Search Engines: Information Retrieval in Practice](https://ciir.cs.umass.edu/irbook) - W. Bruce Croft, Donald Metzler, Trevor Strohman (PDF)
 * [Solr for newbies workshop (2019)](https://github.com/hectorcorrea/solr-for-newbies) - Hector Correa ([PDF](https://github.com/hectorcorrea/solr-for-newbies/blob/master/tutorial.pdf))
+* [SEO: Search Engine Optimization Bible](https://www.pdfdrive.com/seo-search-engine-optimization-bible-e18894055.html) - Jerri L. Ledford
```
Comment (shrn01) [inline books/free-programming-books-subjects.md:735]: Please add official links to the books. PDFDrive is not supported

**Comment by code4oryou** [comment]
updated the link #8154

File: books/free-programming-books-subjects.md [hunk 2]
```diff
@@ -732,6 +732,7 @@ Kerridge (PDF) (email address *requested*, not required)
 * [Elasticsearch: The Definitive Guide](https://www.elastic.co/guide/en/elasticsearch/guide/current/index.html) ([fork it on GH](https://github.com/elastic/elasticsearch-definitive-guide))
 * [Search Engines: Information Retrieval in Practice](https://ciir.cs.umass.edu/irbook) - W. Bruce Croft, Donald Metzler, Trevor Strohman (PDF)
 * [Solr for newbies workshop (2019)](https://github.com/hectorcorrea/solr-for-newbies) - Hector Correa ([PDF](https://github.com/hectorcorrea/solr-for-newbies/blob/master/tutorial.pdf))
+* [SEO: Search Engine Optimization Bible](https://books.google.co.in/books/about/SEO_Search_Engine_Optimization_Bible.html?id=sgmxo1Alq_4C&redir_esc=y) - Jerri L. Ledford
```
Comment (shrn01) [inline books/free-programming-books-subjects.md:735]: Looks like this book is not free

**Pull request closed without merging**